src/main/host/syscall/handler/unistd.rs

```diff
@@ -478,4 +478,11 @@ impl SyscallHandler {
             }
         }
     }
+
+    #[log_syscall(/* rv */ linux_api::posix_types::kernel_pid_t)]
+    pub fn getppid(
+        ctx: &mut SyscallContext,
+    ) -> Result<linux_api::posix_types::kernel_pid_t, SyscallError> {
+        Ok(ctx.objs.process.parent_id().into())
+    }
```

Comment:
Maybe add a small test in `src/test/unistd/test_unistd.rs`? There's a test for `getpid`, so could probably just copy-paste.

Reply:
Done